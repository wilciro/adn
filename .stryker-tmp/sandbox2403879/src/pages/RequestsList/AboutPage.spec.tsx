// @ts-nocheck
import React from 'react';
import { render } from '@testing-library/react';
import RequestedList from '.';

describe('Requested List tests', () => {
  it('should match snapshot AboutPage', () => {
    const { container } = render(<RequestedList />);

    expect(container).toMatchSnapshot();
  });
});
