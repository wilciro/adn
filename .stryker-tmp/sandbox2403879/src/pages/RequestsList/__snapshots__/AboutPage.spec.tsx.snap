// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Requested List tests should match snapshot AboutPage 1`] = `
<div>
  <h2
    id="title-petitions-list"
  >
    Lista de citas
  </h2>
  <div
    class="sc-bczRLJ LwpKF"
  >
    <table
      class="mantine-Table-root __mantine-ref-striped mantine-Table-striped __mantine-ref-hover mantine-Table-hover mantine-1fw2ryq"
    >
      <caption>
        Peticiones
      </caption>
      <thead>
        <tr>
          <th>
            Mascota
          </th>
          <th>
            Tipo mascota
          </th>
          <th>
            Edad mascota
          </th>
          <th>
            Dueño
          </th>
          <th>
            Documento dueño
          </th>
          <th>
            Date
          </th>
          <th>
            Precio
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
